src/components/App/__snapshots__/App.test.js.snap

```diff
@@ -0,0 +1,233 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I think this snapshot is a bit too noisy.
Overall we are not interested in detail what each component at given route contains.
One of the solutions could be mocking them like that:
`jest.mock('../Cart', () => 'Cart')`